tests/nfv/trex_installation.pm

```diff
@@ -39,10 +39,17 @@ sub run {
     assert_script_run("sed -i 's/PORT_1/$PORT_2/' -i $trex_conf");
     assert_script_run("cat $trex_conf");
 
+    # Authorize public key of vsperf machine
+    assert_script_run("mkdir /root/.ssh && chmod 700 /root/.ssh");
```

Comment:
Again, I would prefer creating the keys locally and distributing them instead of having a fixed key distributed

Reply:
how to distribute them inside the test scripts with 2 machines?